packages/transactions/tests/builder.test.ts

```diff
@@ -250,11 +250,23 @@ test('Make Multi-Sig STX token transfer', async () => {
   expect(deserializedPayload.amount.toString()).toBe(amount.toString());
 
   const signer = new TransactionSigner(deserializedTx);
+  // sign once
```

Comment:
Are you modifying an existing test?


Seems like you should be striclty adding tests unless the old tests are not going to pass anymore, which I doubt/ hope not.

Also, I would guess you should add multiple tests.